src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -235,8 +235,9 @@ private static HttpHost getBorderProxy() {
   }
 
   private static String getUserIdUrl(String encodedUsername) {
+    String communityUrl = new UrlBuilder(Configuration.DEFAULT_LANGUAGE).getUrlForWiki("community");
```

Comment:
there is a variable named `COMMUNITY_WIKI` that can be used here.

Reply:
what about other places that try to use UrlBuilder with some preexisting wiki like https://github.com/Wikia/selenium-tests/blob/1c99909b226735ef17a7656b9a4763d6fb169e42/src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditAccountTests.java#L22 or https://github.com/Wikia/selenium-tests/blob/1c99909b226735ef17a7656b9a4763d6fb169e42/src/test/java/com/wikia/webdriver/common/templates/search/IntraWiki.java#L21?

There are a lot of usages of UrlBuilder, each of them created with different params passed to the constructor. It can take a lot of effort to catch all the edge cases.